solutions/cses-1139.mdx

```diff
@@ -68,10 +68,9 @@ struct BIT {
 		}
 };
 
-const int maxn = 2e5+69;
+const int maxn = 2e5+6;
```

Comment:
why +6 now lmao

Reply:
lol y not

Reply:
because when most people look at code they always assume every bit of code is necessary
the problem with this is looking at the +6, they'll assume it's necessary and then be confused about why it is so

Reply:
I mean if they're reading a gold editorial they should be able to infer that it's arbitrary, but for formality ill change it anyways. Just feel that this is a bit nitpicky.